studio-docs/source/sharing-graphs.mdx

```diff
@@ -0,0 +1,498 @@
+
+#### 4. Embed the `iframe`
+
+We're ready to add an embedded Explorer to our page! Here's an example `App.tsx` file that uses our `setupEmbedRelay` function to wire up an embedded Explorer `iframe`:
+
+<MultiCodeBlock>
+
+```tsx:title=App.tsx
+import React, { useEffect } from 'react';
+import { EMBEDDABLE_EXPLORER_URL, setUpEmbedRelay } from './setupEmbedRelay';
+
+export const App = ()=> {
+
+  useEffect(() => {
+    setUpEmbedRelay();
+  }, [])
+
+  // Provide iframe options via URL query parameters
+  const explorerURL = EMBEDDABLE_EXPLORER_URL +
+    '?graphRef=Apollo-Fullstack-Demo-o3tsz8@current' +
+    '&docsPanelState=closed';
+
+  return (
+    <div className="App">
+      <h1 className="demo-app-title">Apollo's Embedded Explorer Demo App</h1>
+      <iframe id="embedded-explorer" className="embedded-explorer-iframe" title="embedded-explorer" src={explorerURL} />
+    </div>
+  );
+}
+```
+
+</MultiCodeBlock>
+
+Notice that we append some query parameters to our `iframe`'s URL, most importantly `graphRef`. This tells the Explorer which [public variant's](#public-graphs) schema to download to populate the Docuentation panel and enable symbol completion.
+
+All supported query parameters are listed in [Options](#options).
+
+#### 5. Run the example
+
+You can view and run a complete example that uses code very similar to the above on Codesandbox:
+
+<a href="https://codesandbox.io/s/github/apollographql/embeddable-explorer-demo-react/tree/main/?fontsize=14&hidenavigation=1&theme=dark">
+  <img alt="Edit embedded-explorer-demo" src="https://codesandbox.io/static/img/play-codesandbox.svg" />
+</a>
+
+The example includes support for subscriptions, which are omitted from this Setup section for clarity. The subscription-specific logic should be more straightforward after familiarizing yourself with the logic for queries and mutations.
+
+### Options
+
+When you embed the Explorer, you set options for it by including query parameters in the `src` URL of the `iframe`, like so:
+
+```jsx:title=EmbeddedExplorer.jsx
+return (
+  <iframe src={`https://explorer.embed.apollographql.com/?graphRef=${myGraphRef}&docsPanelState=closed`} />
+);
+```
+
+The following options are supported:
+
+<table class="field-table api-ref">
+  <thead>
+    <tr>
+      <th>Name /<br/>Type</th>
+      <th>Description</th>
+    </tr>
+  </thead>
+
+<tbody>
+<tr class="required">
+<td>
+
+##### `graphRef`
+
+`string`
+
+</td>
+<td>
+
+**Recommended.** The graph ref for the [public variant](#public-graphs) you want to use the embedded Explorer with. Has the format `graph-name@variant-name`.
+
+The Explorer fetches this variant's schema from Apollo Studio to populate its Documentation panel and enable symbol completion.
+
+If you omit this option, the Explorer still loads, but it can't provide any useful information about the graph the user is querying against.
```

Comment:
@StephenBarlow Maybe we can make this more specific?

`If you omit this option, the Explorer still loads, but it can't provide any useful information about the graph the user is querying against as the Explorer does not know which schema to load. Providing a graph ref helps the Explorer to fetch the schema it needs to load.`

Reply:
Or something more succinct but clarifying that `Explorer does not know what schema to load if you don't do this, SO DO IT!` 😛